File: Tareas 2/Actividad_Abstractos.cpp
#include <string>

class Persona {
private:
    std::string nombre;
    int edad;
    std::string direccion;

public:
    Persona(std::string nombre, int edad, std::string direccion) {
        this->nombre = nombre;
        this->edad = edad;
        this->direccion = direccion;
    }

    // Métodos getter y setter
    std::string getNombre() { return nombre; }
    void setNombre(std::string nombre) { this->nombre = nombre; }
    int getEdad() { return edad; }
    void setEdad(int edad) { this->edad = edad; }
    std::string getDireccion() { return direccion; }
    void setDireccion(std::string direccion) { this->direccion = direccion; }
};

File: Tareas 2/Actividad_Abstractos.cs
public class Persona
{
    private string nombre;
    private int edad;
    private string direccion;

    public Persona(string nombre, int edad, string direccion)
    {
        this.nombre = nombre;
        this.edad = edad;
        this.direccion = direccion;
    }

    // Propiedades
    public string Nombre { get { return nombre; } set { nombre = value; } }
    public int Edad { get { return edad; } set { edad = value; } }
    public string Direccion { get { return direccion; } set { direccion = value; } }
}